Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.py

```diff
@@ -0,0 +1,357 @@
+             }
+    markdown = tableToMarkdown(f'X-Force CVE Reputation for {cve_id}\n'
+                               f'{XFORCE_URL}/vulnerability/search/{cve_id}',
+                               table, removeNull=True)
+
+    return markdown, context, report
+
+
+def test_module(client: Client) -> str:
+    """
+    Returning 'ok' indicates that the integration works like it is supposed to. Connection to the service is successful.
+
+    Args:
+        client (Client): X-Force Exchange client.
+    Returns:
+        str: 'ok' if test passed, anything else will fail the test.
+    """
+
+    return 'ok' if client.ip_report('8.8.8.8')['ip'] == '8.8.8.8' else 'Connection failed.'
+
+
+def ip_command(client: Client, args: Dict[str, str]) -> Tuple[str, dict, dict]:
+    """
+    Executes IP enrichment against X-Force Exchange.
+
+    Args:
+        client (Client): X-Force client.
+        args (Dict[str, str]): the arguments for the command.
+    Returns:
+        str: human readable presentation of the IP report.
+        dict: the results to return into Demisto's context.
+        dict: the raw data from X-Force client (used for debugging).
+    """
+
+    threshold = int(demisto.params().get('ip_threshold', DEFAULT_THRESHOLD))
+    report = client.ip_report(args['ip'])
+
+    outputs = {'Address': report['ip'],
+               'Score': report.get('score'),
+               'Geo': {'Country': report.get('geo', {}).get('country', '')},
+               'Malicious': {'Vendor': 'XFE'}
+               }
+    additional_info = {string_to_context_key(field): report[field] for field in
+                       ['reason', 'reasonDescription', 'subnets']}
+    dbot_score = {'Indicator': report['ip'], 'Type': 'ip', 'Vendor': 'XFE',
+                  'Score': calculate_score(report['score'], threshold)}
+
+    context = {'IP(obj.Address==val.Address)': outputs,
+               'XFE.IP(obj.Address==val.Address)': additional_info,
+               'DBotScore': dbot_score}
+    table = {'Score': report['score'],
+             'Reason': f'{additional_info["Reason"]}:\n{additional_info["Reasondescription"]}',
+             'Subnets': ', '.join(subnet.get('subnet') for subnet in additional_info['Subnets'])}
+    markdown = tableToMarkdown(f'X-Force IP Reputation for: {report["ip"]}\n'
+                               f'{XFORCE_URL}/ip/{report["ip"]}', table, removeNull=True)
+
+    return markdown, context, report
+
+
+def url_command(client: Client, args: Dict[str, str]) -> Tuple[str, dict, dict]:
+    """
+     Executes URL enrichment against X-Force Exchange.
+
+     Args:
+         client (Client): X-Force client.
+         args (Dict[str, str]): the arguments for the command.
+     Returns:
+         str: human readable presentation of the URL report.
+         dict: the results to return into Demisto's context.
+         dict: the raw data from X-Force client (used for debugging).
+     """
+
+    url = args.get('url', '') or args.get('domain', '')

```

Comment:
This will support array of urls/domain. All the reputation commands must support array inputs 
```suggestion
    url = argToList(args.get('url', '')) or argToList(args.get('domain', ''))

```

Reply:
How should array answers be handled, how should the context, output and human readable look like?

Reply:
Just return array, no need to change the contextpath outputs.
```
'URL(val.Name == obj.Name)': [
     {...url_1...},
     {...url_2..}
]
```